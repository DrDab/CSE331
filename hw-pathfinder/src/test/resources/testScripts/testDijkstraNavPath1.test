# Test Dijkstra algo correct

# Create graph g
CreateGraph g

# Add city names to graph g
AddNode g LosSantos
AddNode g SanAndreas
AddNode g SanFierro
AddNode g LasVenturas
AddNode g Ludendorff
AddNode g Vice
AddNode g Liberty
AddNode g Alderney

# Add edges to graph g
AddEdge g LosSantos SanAndreas 50.0
AddEdge g LosSantos SanFierro 300
AddEdge g SanAndreas LosSantos 50
AddEdge g SanAndreas SanFierro 200
AddEdge g SanFierro Ludendorff 1500
AddEdge g Ludendorff Alderney 2000
AddEdge g Alderney Liberty 30

# Test pathfinding
FindPath g LosSantos Liberty
