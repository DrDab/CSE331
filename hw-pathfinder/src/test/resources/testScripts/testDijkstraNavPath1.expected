# Test Dijkstra algo correct

# Create graph g
created graph g

# Add city names to graph g
added node LosSantos to g
added node SanAndreas to g
added node SanFierro to g
added node LasVenturas to g
added node Ludendorff to g
added node Vice to g
added node Liberty to g
added node Alderney to g

# Add edges to graph g
added edge 50.000 from LosSantos to SanAndreas in g
added edge 300.000 from LosSantos to SanFierro in g
added edge 50.000 from SanAndreas to LosSantos in g
added edge 200.000 from SanAndreas to SanFierro in g
added edge 1500.000 from SanFierro to Ludendorff in g
added edge 2000.000 from Ludendorff to Alderney in g
added edge 30.000 from Alderney to Liberty in g

# Test pathfinding
path from LosSantos to Liberty:
LosSantos to SanAndreas with weight 50.000
SanAndreas to SanFierro with weight 200.000
SanFierro to Ludendorff with weight 1500.000
Ludendorff to Alderney with weight 2000.000
Alderney to Liberty with weight 30.000
total cost: 3780.000
